src/EnergyPlus/DaylightingManager.cc

```diff
@@ -1846,20 +1744,23 @@ void FigureDayltgCoeffsAtPointsSetupForWindow(
 
             //            ! Error message if ref pt is too close to window.
             if (D1a > 0.0 && D1b > 0.0 && D1b <= HW && D1a <= WW) {
-                ShowSevereError(state,
-                                format("CalcDaylightCoeffRefPoints: Daylighting calculation cannot be done for zone {} because reference point "
-                                       "#{} is less than 0.15m (6\") from window plane {}",
-                                       Zone(ZoneNum).Name,
-                                       iRefPoint,
-                                       state.dataSurface->Surface(IWin).Name));
+                ShowSevereError(
+                    state,
+                    format("CalcDaylightCoeffRefPoints: Daylighting calculation cannot be done for Daylighting:Controls={} because reference point "
+                           "#{} is less than 0.15m (6\") from window plane {}",
+                           state.dataDaylightingData->daylightControl(daylightCtrlNum).Name,
+                           iRefPoint,
+                           state.dataSurface->Surface(IWin).Name));
                 ShowContinueError(state, format("Distance=[{:.5R}]. This is too close; check position of reference point.", ALF));
                 ShowFatalError(state, "Program terminates due to preceding condition.");
             }
         } else if (ALF < 0.1524 && ExtWinType == DataDaylighting::iExtWinType::AdjZoneExtWin) {
             if (state.dataDaylightingManager->RefErrIndex(iRefPoint, IWin) == 0) { // only show error message once
                 ShowWarningError(state,
-                                 "CalcDaylightCoeffRefPoints: For Zone=\"" + Zone(ZoneNum).Name + "\" External Window=\"" +
-                                     state.dataSurface->Surface(IWin).Name + "\"in Zone=\"" + Zone(state.dataSurface->Surface(IWin).Zone).Name +
+                                 "CalcDaylightCoeffRefPoints: For Daylghting:Controls=\"" +
+                                     state.dataDaylightingData->daylightControl(daylightCtrlNum).Name + "\" External Window=\"" +
+                                     state.dataSurface->Surface(IWin).Name + "\"in Zone=\"" +
+                                     state.dataHeatBal->Zone(state.dataSurface->Surface(IWin).Zone).Name +
```

Comment:
I see. You changed from zone to daylighting controls. So this is also OK, just a different/better reference to where the problem is.